package.json

```diff
@@ -153,10 +154,48 @@
     "@babel/plugin-proposal-private-methods": "^7.14.5",
     "@babel/plugin-proposal-private-property-in-object": "^7.14.5",
     "@babel/runtime-corejs2": "^7.14.8",
-    "@changesets/cli": "^2.14.1",
+    "@changesets/cli": "^2.26.0",
     "legally": "^3.5.10",
     "pascal-case": "^3.1.2",
     "react-json-tree": "^0.11.2",
     "shelljs": "^0.8.3"
+  },
+  "resolutions": {
```

Comment:
Wow, I'm curious about how you generated this list? It's been a painful process to determine which internal dependencies are depended on by our project's dependencies, and this is a big list, did you use a helpful tool?

Reply:
Unfortunately, I don't have a helpful tool 😄 I used `yarn audit` to determine which package needs an upgrade, and checked `yarn.lock` so that it does not introduce a breaking change.